src/test/java/org/dependencytrack/resources/v1/TagResourceTest.java

```diff
@@ -130,13 +131,15 @@ public void getTagsTest() {
                     "name": "bar",
                     "projectCount": 1,
                     "policyCount": 1,
-                    "notificationRuleCount": 0
+                    "notificationRuleCount": 0,
+                    "vulnerabilityCount": 0
                   },
                   {
                     "name": "foo",
                     "projectCount": 2,
                     "policyCount": 0,
-                    "notificationRuleCount": 1
+                    "notificationRuleCount": 1,
+                    "vulnerabilityCount": 0
                   }
```

Comment:
There should be another tag that has a vulnerability assigned to it for this test.